codalab/apps/web/static/js/worksheet/worksheet_interface.js

```diff
@@ -344,6 +344,7 @@ var Worksheet = React.createClass({
                     subFocusIndex={this.state.subFocusIndex}
                     myHomeWorksheet={this.myHomeWorksheet}
                     uploadBundle={this.uploadBundle}
+                    updateContent={this.refreshWorksheet}
```

Comment:
Any particular reason we can't call this `refreshWorksheet` as well so we can keep names consistent?